### PR TITLE
Smashing Magazine Copy Heat Map

### PR DESCRIPTION
**Pull request by atoledo-ec**
Content:
- Have the main page on Smashing Magazine in Gray scale coloured to show hierarchy
- Used flexbox to position elements

### CODE REVIEW

File: .vscode/settings.json
Comment (teekaytech) [inline .vscode/settings.json]: This file is irrelevant to the project. Kindly remove it.

File: README.md
Comment (teekaytech) [inline README.md]: Your live demo link is broken. Kindly fix it.

File: index.html
Comment (teekaytech) [inline index.html]: These items should be linked.

File: index.html
Comment (teekaytech) [inline index.html]: As your class name implies, this looks like an article and should be treated as such. Please use the `<article>` tag.

File: index.html
Comment (teekaytech) [inline index.html]: This is another article. Kindly wrap __this and all other articles__ inside the `<article>` tag.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: 

### Hi team :wave: 

**I must commend your efforts in this project. So far, the project is looking good. But we have to make some changes before this project can be approved. Kindly check the issues I raised below and make changes as required.**

**[ REQUIRED CHANGES ]**

- [x] You have a folder `.vscode` that is not related to the project. Always add unrelated files/folders used in development to the `.gititnore` file so that it doesn't get committed. Kindly find a way to remove the folder.



- [x] You used the `<div>` tag for all your articles. Kindly use the `<article>` tag for all your articles in this project. 



- [x] Check the image below and fix the overlapping.



- [x] Your footer items should be linked. See the image below as reference.



**Please make the required changes and request for another review when you are done.**

__I'm @taofeekolalere on slack.
Happy Coding :smile:__

**Review by ghost — CHANGES_REQUESTED** [review]
# Status: Changes Required

## Hello team,

**Your project looks good. The only issue is that your linter checks are not working. Please set up the linter checks as required. You can follow this [guideline](https://github.com/microverseinc/linters-config/tree/master/html-css). Also,` .stylelintrc.json` and `.hintrc` files are very much useful to the project and should therefore not be added to the `.gitignore` file. Github action uses these files to run linter checks.**

### Happy coding 😃. 

**PS: You can reach me on slack(Rose Wanjohi)**

**Review by ghost — APPROVED** [review]
# Status: Approved

**Congratulations team and good luck on your next project 😃.**

**Pull request merged**
